examples/api/service/client.go

```diff
@@ -10,20 +11,22 @@ import (
 	peer "github.com/ipfs/go-ipfs/p2p/peer"
 	fsrepo "github.com/ipfs/go-ipfs/repo/fsrepo"
 
-	"golang.org/x/net/context"
```

Comment:
we should be keeping
```
"golang.org/x/net/context"
```
as that's the official address. `code.google.com` is deprecated.